content/200-concepts/150-data-platform/01-about-platform/100-platform-limits-and-status.mdx

```diff
@@ -0,0 +1,84 @@
+---
+
+<TopBlock>
+
+On this page, you can find information about the limits that the Prisma Data Platform and the Prisma Data Proxy have in place.
+
+You can also learn about the compatibility of the platform with the Prisma ORM and the availability and SLOs that the platform provides.
+
+</TopBlock>
+
+## General limits in the Prisma Data Platform
+
+The table below provides a list of general limits that the Prisma Data Platform has in place.
+
+<Admonition>
+
+**Extending general limits**<br /><br />
+Contact Support if you need to have any of the existing limits lifted for your project.
+
+</Admonition>
+
+| Limit type                                                                                                                                                            | Value          |
+| --------------------------------------------------------------------------------------------------------------------------------------------------------------------- | -------------- |
+| Maximum number of projects on the **Free** plan per account<br /><br /><br />**Note**<br /><br /> The number does not include projects on the **Early Adopter** plan. | 10             |
+| Maximum number of projects per account                                                                                                                                | 100            |
+| Maximum number of collaborators per project                                                                                                                           | 100            |
+| Maximum number of environments per project                                                                                                                            | 100            |
+| Maximum number of connection strings per environment                                                                                                                  | 100            |
+| Response timeout in Data Browser                                                                                                                                      | 60s            |
+| Response timeout in Query Console                                                                                                                                     | 60s            |
+| Maximum number of rows fetched in Data Browser or Query Console                                                                                                       | Not applicable |
+
+## Concurrency limits of the Prisma Data Proxy
+
+The concurrency limits of the Prisma Data Proxy define the maximum number of proxy instances that can run simultaneously for a single project. By default, only one proxy instance is started for a single project environment.
```

Comment:
It might be worth noting here that this scales up when needed. Also, the one instance isn't strictly true - we spin up at least one, in two distinct failure domains.

Reply:
Changed to "at least two proxy instances are started...". Is that enough, I mean it doesn't sound necessary to also add "two distinct failure domains", right?

Reply:
What I'm trying to get at here is that you can get lucky end up with only one instance, but if there's enough traffic coming from your end, you will most likely end up with two.

Reply:
This is changed for now to "at least one proxy instance".

Resolving for now. And let's revisit after GA if we still feel like it makes sense.